src/app/app-platform/ContentApp.h

```diff
@@ -94,11 +95,32 @@ class ContentAppClientCommandSender
 class DLL_EXPORT ContentApp
 {
 public:
+    struct SupportedCluster
+    {
+        chip::ClusterId mClusterIdentifier{ chip::kInvalidClusterId };
```

Comment:
Note: this is all inside `namespace chip`, so all these `chip::` are probably just noise.

Reply:
Updated!